v2/pkg/reporting/trackers/jira/jira.go

```diff
@@ -46,6 +46,8 @@ type Options struct {
 	HttpClient      *retryablehttp.Client
 }
 
+const coderegex = "\n{code}\n"
```

Comment:
Please choose a better name, and a follow the naming conventions.

Reply:
Done